SimPEG/electromagnetics/static/induced_polarization/simulation.py

```diff
@@ -7,35 +7,27 @@
 from ....utils import mkvc, sdiag
 from ...base import BaseEMSimulation
 
+from ..resistivity.simulation import BaseDCSimulation
 from ..resistivity.fields import FieldsDC, Fields3DCellCentered, Fields3DNodal
 from ..resistivity import Simulation3DCellCentered as BaseSimulation3DCellCentered
 from ..resistivity import Simulation3DNodal as BaseSimulation3DNodal
 
 
-class BaseIPSimulation(BaseEMSimulation):
-
-    sigma = props.PhysicalProperty("Electrical conductivity (S/m)")
-
-    rho = props.PhysicalProperty("Electrical resistivity (Ohm m)")
-
-    props.Reciprocal(sigma, rho)
+class BaseIPSimulation(BaseDCSimulation):
 
     eta, etaMap, etaDeriv = props.Invertible("Electrical Chargeability")
 
     data_type = properties.StringChoice(
```

Comment:
@jcapriot: do we need this in the simulation class?

Reply:
We certainly do not, but I left it there for backwards compatibility. I should probably add a deprecation warning saying to define it on the receiver.